src/dotnet/commands/dotnet-pack/NuGet/Manifest.cs

```diff
@@ -8,7 +8,7 @@
 using System.Linq;
 using System.Xml.Linq;
 
-namespace NuGet
+namespace NuGet.Legacy
```

Comment:
Is this namespace change necessary? Won't we be removing this code in the near future?